CWE-312/README.md

```diff
@@ -0,0 +1,102 @@
+# Detect CWE-312 in Android Application (ovaa.apk)
+
+This scenario seeks to find cleartext storage of sensitive data in the APK file. See [CWE-312](https://cwe.mitre.org/data/definitions/312.html) for more details.
+
+Let’s use this [APK](https://github.com/oversecured/ovaa) and the above APIs to show how Quark script find this vulnerability.
+
+First, we designed a Frida script agent.js to hook the target method and get the arguments when the target method is called. Then we hook the method putString to catch its arguments. Finally, we use Ciphey to check if the arguments are encrypted.
```

Comment:
The links to the official websites of Frida and Ciphey is missing. Please correct it.